schema.c

```diff
@@ -1131,6 +1131,42 @@ sch_lookup (sch_instance * instance, const char *path)
     return lookup_node (instance, NULL, xmlDocGetRootElement (instance->doc), path);
 }
 
+void
+sch_lookup_node_get_ns_info (sch_instance * instance, const char *path, char **href, char **prefix)
```

Comment:
We already have sch_namespace that returns the href ... I am not sure the model prefix is useful as in netconf the prefix will be defined in the XML data .. but if you need to get the default prefix from the model then we can create a new function called sch_prefix that returns a strdup'ed default prefix for the model

Reply:
I have created the new function sch_prefix and removed the function sch_lookup_node_get_ns_info